transformations/chinese_butter_fingers_perturbation/transformation.py

```diff
@@ -0,0 +1,80 @@
+import itertools
+import random
+import unidecode as u
+import json
+import os
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+def butter_finger(text, prob=0.2, keyboard="pinyin", seed=0, max_outputs=1):
+    random.seed(seed)
+
+    prob_of_typo = prob * 100
+    perturbed_texts = []
+    for _ in itertools.repeat(None, max_outputs):
+        butter_text = ""
+        for chinese_character in text:
+            similar_pinyins = get_characters_with_similar_pinyin(chinese_character, keyboard)
+
+            if random.choice(range(0, 100)) <= prob_of_typo and similar_pinyins != "":
```

Comment:
`random.random() <= prob` would be equivalent, and you don't need to multiply prob by 100.

Reply:
Agreed and Addressed.